engineering/continuous-delivery.md

```diff
@@ -0,0 +1,93 @@
+    - [Continuous delivery](#continuous-delivery)
+- [General workflow](#general-workflow)
+- [Deploying into Production](#deploying-into-production)
+    - [special cases](#special-cases)
+        - [Release branch](#release-branch)
+        - [Exceptions](#exceptions)
+- [Deploying into Staging](#deploying-into-staging)
+
+<!-- /TOC -->
+
+# Intro
+
+This documentation assumes it is already known the [company deployment workflow](git-flow.md), and extends it to cover the continuous delivery as it should work for all source{d} web applications:
+
+repo | Prod | Staging
+--- | --- | ---
+[src-d/landing](https://github.com/src-d/landing) | [sourced.tech](https://sourced.tech) | [landing-staging.srcd.run](https://landing-staging.srcd.run)
+[src-d/blog](https://github.com/src-d/blog) | [blog.sourced.tech](http://blog.sourced.tech) | [blog-staging.srcd.run](https://blog-staging.srcd.run)
+[src-d/talks](https://github.com/src-d/talks) | [talks.sourced.tech](https://talks.sourced.tech) | [talks-staging.srcd.run](https://talks-staging.srcd.run)
+
+This document does not pretend to be exhaustive but a guide, and there can be special cases that will be defined in the README.md and CONTRIBUTING.md of each web application repository.
+
+This document is based on the agreements of the [2017-10-23 CD workflow](https://github.com/src-d/minutes/blob/master/infrastructure/2017-10-23-landing-cd-workflow.md) and [2017-10-26 Enforcing the CD workflow](https://github.com/src-d/minutes/blob/master/infrastructure/2017-10-26-enforcing-cd-workflow.md) meetings.
+
+# Quick summary:
+
+Here are the technical details of when CI or CD runs, but it is needed you know and understand the guides and conventions explained below this section.
+
+## branches
+- `master` branch must be always production ready and will reflect the code deployed into Production.
+- there are no long-term branches but `master`,
+- `staging` branch must not be merged into other branches,
+- `staging` branch should be kept as close to master as possible,
+
+## Continuous delivery
+CI and CD is controlled [by drone](https://drone.srcd.host)
+
+- It is automatically deployed into Production when something is tagged,
+- it is automatically deployed into Staging when the `src-d:staging` branch is updated,
+- CI automatically runs when a PR against `master` or a `release/*` branch is created or updated with new commits.
+
+
+# General workflow
+
+`master` branch must be "always" production ready.
+
+`master` branch will contain only approved commits through a PR.
+
+`master` branch is the starting point for feature branches (considering some [special cases](#special-cases))
+
+Since the workflow in the source{d} web applications use to be closely guided by Product, it is quite likely that it is needed to obtain its approval before merging into `master`; the repository [Maintainer](maintainers.md) will decide if any PR must be reviewed by Product, by itself or by any other person.
```

Comment:
I don't think this is better! `itself` can never refer to people.

Reply:
Yep, "itself" is used for things, not people. Quite common nowadays to use plural for gender-neutral language, so "themself" would fit well. More background info on this: https://blog.oxforddictionaries.com/2013/01/15/themself/